entries/e/efirstbank.com.json

```diff
@@ -2,11 +2,17 @@
   "FirstBank": {
     "domain": "efirstbank.com",
     "url": "https://www.efirstbank.com/",
-    "contact": {
-      "email": "banking@efirstbank.com",
-      "facebook": "efirstbank",
-      "twitter": "efirstbank"
-    },
+    "tfa": [
+      "custom-software",
+      "custom-hardware"
+    ],
+    "custom-software": [
+      "FirstBank App Push Notification"
+    ],
+    "custom-hardware": [
+      "Only available for commerical users"
```

Comment:
1. The value here should be the name of the hardware used. Any restrictions can be displayed as a note.
2. I don't see any documentation for hardware 2FA on efirstbank.com.

Reply:
Apologies, I had to go try and track down some of this information. I couldn't find any specific information about which hardware token is being used, other than the website just mentioning that tokens are used: https://efirstbank.com/customer-service/questions-answers.htm#internetCashManagement (the section labeled "Is Internet Cash Management secure"). The tokens are only offered to commercial users who opt in for the token and use them during login and Bill Pay. I will change the value to just be "hardware" instead, if that works.